content/en/docs/contribute/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+content_template: templates/concept
+title: Contribute to Kubernetes docs
+linktitle: Contribute
+main_menu: true
+weight: 80
+---
+
+{{% capture overview %}}
+
+If you would like to help contribute to the Kubernetes documentation or website,
```

Comment:
A small nit:
The left nav item for `start contributing` is just `start`. How about 'Getting Started', or a different way to say, beginner?

Reply:
It's under a Contribute heading so I thought it would be fine to be concise. I wanted a strong call-to-action.

Reply:
Update to say I addressed this by changing it to "Start contributing".